src/components/assetlib_projects/asset_library_projects.tscn

```diff
@@ -0,0 +1,223 @@
+[gd_scene load_steps=2 format=3 uid="uid://c1sh1x6y1kwc2"]
+
+[ext_resource type="Script" path="res://src/components/assetlib_projects/asset_library_projects.gd" id="1_m0pc7"]
+
+[node name="AssetLibraryProjects" type="VBoxContainer"]
+anchors_preset = 15
+anchor_right = 1.0
+anchor_bottom = 1.0
+grow_horizontal = 2
+grow_vertical = 2
+script = ExtResource("1_m0pc7")
+
+[node name="SearchField" type="LineEdit" parent="."]
+unique_name_in_owner = true
+layout_mode = 2
+placeholder_text = "Search Templates, Projects, and Demos"
+
+[node name="FilterSortOptions" type="HBoxContainer" parent="."]
+layout_mode = 2
+
+[node name="Label4" type="Label" parent="FilterSortOptions"]
+layout_mode = 2
+text = "Version:"
+
+[node name="VersionOption" type="OptionButton" parent="FilterSortOptions"]
+unique_name_in_owner = true
+layout_mode = 2
+size_flags_horizontal = 3
+
+[node name="VSeparator4" type="VSeparator" parent="FilterSortOptions"]
+layout_mode = 2
+
+[node name="Label" type="Label" parent="FilterSortOptions"]
+layout_mode = 2
+text = "Sort:"
+
+[node name="SortOption" type="OptionButton" parent="FilterSortOptions"]
+unique_name_in_owner = true
+layout_mode = 2
+size_flags_horizontal = 3
+item_count = 6
+selected = 0
```

Comment:
please add clip_text = true for this and any other OptionButton in the HBoxContainer in order to allow it to be fit correctly when window is resized to its minimum size